File: MoliStudy/Subject.m
//
//  Subject.m
//  MoliStudy1
//
//  Created by zhaoqin on 10/25/15.
//  Copyright © 2015 张鹏. All rights reserved.
//

#import "Subject.h"

@implementation Subject

- (void) initData{
    self.content = [[NSMutableArray alloc] init];
    self.answers = [[NSMutableArray alloc] init];
    self.correctAnswer = [[NSString alloc] init];
    self.thinkLabelID = [[NSMutableArray alloc] init];
    self.thinkLabel = [[NSMutableArray alloc] init];
    self.questionID = [[NSString alloc] init];
}

@end

File: MoliStudy/Subject.swift
//
//  Subject.swift
//  MoliStudy
//
//  Created by zhaoqin on 11/30/15.
//  Copyright © 2015 MoliStudy. All rights reserved.
//

import UIKit

class Subject: NSObject {

    var content: [String]
    var answers: [[String]]
    var correctAnswer: NSInteger
    var questionID: NSNumber
    var thinkLabel: [ThinkLabel]
    var allString: [String]
    
    override init() {
        content = []
        answers = []
        correctAnswer = -1
        questionID = NSNumber()
        thinkLabel = []
        allString = []
    }
    
    
}
